TicTacToeApp/ViewController.swift

```diff
@@ -1,152 +1,99 @@
 
 import UIKit
 
-public class BoardButton: UIButton {
-}
-
-public class BoardView: UIView {
-
-  // hypothetical interface
-  public func enableInput() {}
-  public func disableInput() {}
-  public func clear() {}
-
-}
-
-// Refactoring alert dialog
-// Might be overkill. Let's revisit after board button refactorings
-//public class ConfirmResetGameAlert: UIAlertController {}
-// Alternatively, consider builder patter, or just wrapping the alert dialog
-//     i.e. composition over inheritance
-//public class ConfirmResetGameAlert {
-//  var alertController: UIAlertController?
-//}
-
 public class ViewController: UIViewController {
```

Comment:
That makes sense. Is it helpful to keep "View" in the name at all? We're thinking something like GameViewController or GameplayController. We'd certainly need to keep the names descriptive if we added a second ViewController. A second controller would probably help with naming by contrasting their functions.